hawkauthlib/tests/__init__.py

```diff
@@ -1,3 +1,11 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+
+try:
+    import os
+    if os.environ.get("DEBUG", None):
+        from pytest import set_trace
+        __builtins__["DEBUG"] = set_trace
```

Comment:
I'm not sold on this; altering builtins based on the environment seems risky, especially for a library that can't know what its calling application expects environment variables to mean.  If you feel stringly about keeping it, perhaps we can call the env var "PYTEST_DEBUG" or something a little more specific?

Reply:
It is for the `make debug` target - run tests within a PDB debug session. For me, in some cases it was helpful but if you don't like it, I can remove the make target and the py-implementation. Renaming PYTEST_DEBUG is also a good suggestion .. it is up to you.

Reply:
My default position would be to remove it, but if you think it'll be helpful in future then we can keep it under a more explicit name `PYTEST_DEBUG`.  I'll leave the final call on that up to you.

Reply:
To be more MVP, I decided to removed the debug quirks from the boilerplate.